configs/default/DAKE-DAKEFPVF405.config

```diff
@@ -12,6 +12,8 @@
 #define USE_BARO
 #define USE_BARO_SPI_BMP280
 #define USE_BARO_SPI_DPS310
+#define USE_MAG_HMC5883
+#define USE_MAG_QMC5883
```

Comment:
Does the flight controller include the magnetometer on board ?

Reply:
> Does the flight controller include the magnetometer on board ?

No, the board has SDA and SCL pads for connecting the compass.

Reply:
Okay in that case they need to be removed. They will be included when user adds magnetometer option during cloud build.